### PR TITLE
Add Alert if selected University is Sorbonne or Sorbonne Paris Nord

### PR DESCRIPTION
**Pull request by Evidorus**

### CODE REVIEW

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx [hunk 1]
```diff
@@ -35,6 +35,9 @@ const ConventionForm = ({ currentConvention, onConventionUpdated, checkDefaultVa
       });
   };
 
+  const laSorbonne = 'b394d7e8-a968-53f7-9016-e37d070080b0';
+  const laSorbonneParisNord = '47ec8346-6019-588e-af40-2c437361ecac';
+
```
Comment (desoindx) [inline frontend/src/components/Psychologist/Profile/ConventionForm.jsx:40]: that's not the prod uuid...
Would be nice to get all the universities  from the back and find the one by name ?

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx [hunk 2]
```diff
@@ -56,6 +59,14 @@ const ConventionForm = ({ currentConvention, onConventionUpdated, checkDefaultVa
               ? universities.map(university => ({ value: university.id, label: university.name }))
               : []}
           />
+          {(convention.universityId === laSorbonneParisNord || convention.universityId === laSorbonne) && (
+            <Alert
+              description="Pensez à vérifier que l'université sélectionnée est exacte.
+            Dans le cas de La Sorbonne (75) et Sorbonne Paris Nord (93)."
```
Comment (desoindx) [inline frontend/src/components/Psychologist/Profile/ConventionForm.jsx:65]: je preferais le wording de Sandrine mais a checker ?
```suggestion
            Il y a La Sorbonne (75) et Sorbonne Paris Nord (93)."
```

Comment (Evidorus) [reply frontend/src/components/Psychologist/Profile/ConventionForm.jsx:65]: Justement Sandrine a conseillé de prendre celui de Lina.

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx [hunk 3]
```diff
@@ -56,6 +59,15 @@ const ConventionForm = ({ currentConvention, onConventionUpdated, checkDefaultVa
               ? universities.map(university => ({ value: university.id, label: university.name }))
               : []}
           />
+          {universities.length >= 1
+          && (convention.universityId === laSorbonneParisNord.id || convention.universityId === laSorbonne.id) && (
```
Comment (desoindx) [inline frontend/src/components/Psychologist/Profile/ConventionForm.jsx:63]: not the correct if
```suggestion
          {(laSorbonneParisNord && convention.universityId === laSorbonneParisNord.id) || (laSorbonne && convention.universityId === laSorbonne.id)) && (
```

**Review by desoindx — CHANGES_REQUESTED** [review]

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx (hunk 1 above)
Comment (sandrica89) [reply frontend/src/components/Psychologist/Profile/ConventionForm.jsx]: :thinking: why not compare by name?

Comment (desoindx) [reply frontend/src/components/Psychologist/Profile/ConventionForm.jsx]: that's what he did ?

**Review by desoindx — APPROVED** [review]

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx (hunk 1 above)
Comment (Evidorus) [reply frontend/src/components/Psychologist/Profile/ConventionForm.jsx]: Yep i did it already

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx [hunk 4]
```diff
@@ -56,6 +59,13 @@ const ConventionForm = ({ currentConvention, onConventionUpdated, checkDefaultVa
               ? universities.map(university => ({ value: university.id, label: university.name }))
               : []}
           />
+          {((laSorbonneParisNord && convention.universityId === laSorbonneParisNord.id) || (laSorbonne && convention.universityId === laSorbonne.id)) && (
```
Comment (sandrica89) [inline frontend/src/components/Psychologist/Profile/ConventionForm.jsx:62]: I would have extract this part in a const called for example `isSorbonne`

**Review by sandrica89 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx (hunk 4 above)
Comment (Evidorus) [reply frontend/src/components/Psychologist/Profile/ConventionForm.jsx:62]: I don't think it's the best option, everywhere we did it directly in the return. Maybe if we do this here, we have to do it everywhere then :) But if u think it's more visible and efficient, i will do it 👍

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx (hunk 1 above)
Comment (sandrica89) [reply frontend/src/components/Psychologist/Profile/ConventionForm.jsx]: Oops! I didn't understand it right, sorry!

File: frontend/src/components/Psychologist/Profile/ConventionForm.jsx (hunk 4 above)
Comment (Evidorus) [reply frontend/src/components/Psychologist/Profile/ConventionForm.jsx:62]: we discuss ( she gave me good arguments ) and i will do it :)

Comment (Evidorus) [reply frontend/src/components/Psychologist/Profile/ConventionForm.jsx]: Done 🥇

**Comment by sandrica89** [comment]
- [ ] Add some space below the alert
- [ ] Bonus: in convention modal, add some space before convention form

**Comment by Evidorus** [comment]
I added the space for the Alert but for the bonus I give up for now ^^'

**Comment by desoindx** [comment]
> I added the space for the Alert but for the bonus I give up for now ^^'

in this case create a trello card

**Comment by sandrica89** [comment]
> > I added the space for the Alert but for the bonus I give up for now ^^'
> 
> in this case create a trello card
Done

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**